Jenkinsfile.github

```diff
@@ -58,47 +58,56 @@ pipeline {
     triggers { cron('H 16 * * 0') }
 
     environment {
-        BUILD_ARGS = "--build-arg 'SLE_VERSION=${sleVersion}' --build-arg 'PY_VERSION=${pythonVersion}' --secret id=SLES_REGISTRATION_CODE"
-        DOCKER_ARGS = getDockerBuildArgs(name: getRepoName(), description: 'A build environment for Python.')
-        DOCKER_BUILDKIT = 1
-        MULTI_ARCH = 1
         NAME = getRepoName()
         PY_VERSION = "${pythonVersion}"
-        SLE_VERSION = "${sleVersion}"
         SLES_REGISTRATION_CODE = credentials('sles15-registration-code')
         TIMESTAMP = sh(returnStdout: true, script: "date '+%Y%m%d%H%M%S'").trim()
         VERSION = "${GIT_COMMIT[0..6]}"
+        DOCKER_BUILDKIT = 1
+        MULTI_ARCH = 1
     }
 
     stages {
+        stage('Build and Publish') {
+            matrix {
+                axes {
+                    axis {
+                        name 'SLE_VERSION'
+                        values '15.5', '15.6'
+                    }
+                }
+                stages {
+                    stage('Build') {
+                        steps {
+                            script {
+                                env.BUILD_ARGS = "--build-arg SLE_VERSION=${SLE_VERSION} --build-arg PY_VERSION=${PY_VERSION} --secret id=SLES_REGISTRATION_CODE"
+                                sh "make image"
+                            }
+                        }
+                    }
+                    stage('Publish') {
+                        steps {
+                            script {
 
-        stage('Build') {
-
-            steps {
-                sh "make image"
-            }
-        }
-
-        stage('Publish') {
-            steps {
-                script {
-
-                    // Only overwrite an image if this is a stable build.
-                    if (isStable) {
-                        /*
-                        Publish these tags on stable:
-                            - Major.Minor                          (e.g. 3.10)
-                        */
-                        publishCsmDockerImage(image: env.NAME, multiArch: env.MULTI_ARCH, isStable: isStable, tag: "${pythonVersion}")
-                        publishCsmDockerImage(image: env.NAME, multiArch: env.MULTI_ARCH, isStable: isStable, tag: "latest")
+                                // Only overwrite an image if this is a stable build.
+                                if (isStable) {
```

Comment:
and only if the branch is `main`.

Reply:
You may need to add this change to main and every other `maint/` branch in this repo, a repeat of MTL-2408.